warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -10270,4 +10270,8 @@ public static double kurtosis(GeoTimeSerie gts, boolean bessel) throws WarpScrip
   public static double skewness(GeoTimeSerie gts, boolean bessel) throws WarpScriptException {
     return standardizedMoment(3, gts, bessel);
   }
+
+  public static void booleanNot(GeoTimeSerie gts) {
+    gts.booleanValues.flip(0, gts.booleanValues.length());
```

Comment:
Should check that the GTS is indeed a boolean one, otherwise gts.booleanValues might be null and this could lead to a NPE